client/client.go

```diff
@@ -12,24 +12,29 @@ import (
 	"time"
 )
 
-var logger = log.ClientLogger
+var Logger log.Logger
```

Comment:
Is there any reason why we're now exporting the logger? Probably I'm just missing something.